src/uhttp.c

```diff
@@ -187,30 +187,39 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
             colonEncountered = true;
             size_t keyLen = (&buffer[index])-targetPos;
```

Comment:
nit: space this out if you can :)
```suggestion
            size_t keyLen = (&buffer[index]) - targetPos;
```